review_code_gemini.py

```diff
@@ -60,61 +60,53 @@ def analyze_code(parsed_diff: List[Dict[str, Any]], pr_details: PRDetails) -> Li
     """Analyzes the code changes using Gemini and generates review comments."""
     print("Starting analyze_code...")
     print(f"Number of files to analyze: {len(parsed_diff)}")
-    #print(f"Files content: {json.dumps(parsed_diff, indent=2)}")
     comments = []
-    print(f"Initial comments list: {comments}")
+    #print(f"Initial comments list: {comments}")
     
     for file_data in parsed_diff:
         file_path = file_data.get('path', '')
```

Comment:
Using a class `FileInfo` for just storing the file path seems overkill.  Consider simplifying this to just use the `file_path` string directly.